app/components/mandatarissen/mandataris-card.hbs

```diff
@@ -1,116 +1,130 @@
-<AuCard as |c|>
-  <c.header>
-    <div class="au-u-flex">
-      <AuHeading @level="2" @skin="4" class="flex-grow">
-        {{this.rol}}
-      </AuHeading>
-      <AuPill
-        @skin="link"
-        @draft={{false}}
-        @size="regular"
-        @icon="info-circle"
-        @iconAlignment="left"
-        @hideText={{false}}
-      >
-        {{if
-          @mandataris.publicationStatus
-          @mandataris.publicationStatus.label
-          "Bekrachtigd"
-        }}
-      </AuPill>
-    </div>
-  </c.header>
-  <c.content>
+<AuToolbar @size="large" class="au-u-padding-none au-u-bottom-none" as |Group|>
+  <Group>
+    <AuHeading @skin="4">
+      Details mandaat
+    </AuHeading>
+  </Group>
+</AuToolbar>
 
-    <div class="au-o-grid au-o-grid--tiny">
-      <div class="au-o-grid__item au-u-1-2 au-u-1-3@medium">
-        <div class="au-c-description-label">Status</div>
-        <div class="au-c-description-value">{{this.status}}</div>
-      </div>
-      <div class="au-o-grid__item au-u-1-2 au-u-1-3@medium">
-        <div class="au-c-description-label">Rangorde</div>
-        <div class="au-c-description-value">{{@mandataris.rangorde}}</div>
-      </div>
-      <div class="au-o-grid__item au-u-1-2 au-u-1-3@medium">
-        <div class="au-c-description-label">Fractie</div>
-        <div class="au-c-description-value"><Mandaat::FoldedFracties
-            @persoon={{this.persoon}}
-            @bestuursperiode={{@bestuursperiode}}
-          /></div>
+<div class="au-o-box au-u-background-gray-100">
+  <div class="au-o-grid au-o-grid--tiny">
+    <div class="au-o-grid__item au-u-1-2">
+      <div class="au-c-description-label">Status</div>
+      <div
+        class="au-c-description-value au-u-margin-top-tiny"
```

Comment:
design has this as a pill too

Reply:
Updated it. Doe sit need extra logic for colors?

Reply:
Effectief status will be green the rest is gray